Readme.md

```diff
@@ -1,168 +1,185 @@
 
 ![Blazor - Theme Switcher](images/blazor-theme-switcher.png)
 
-The example's solution targets .NET 8, but you can also integrate this Theme Switcher in projects that target .NET 6 and .NET 7.
+## Configure Available Themes
 
-## Add a Theme Switcher to an Application
+The theme switcher includes the following themes:
+
+* DevExpress Fluent (Light Blue and Dark Blue)
+* DevExpress Classic (Blazing Berry, Blazing Dark, Purple, and Office White)
+* [Bootstrap External](https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css)
+
+
+Create a `Themes.cs` file and configure themes as follows:
+
+1. For Classic themes, choose a theme from the built-in DevExpress Blazor [Themes](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Themes) collection:
 
-Follow the steps below to add a Theme Switcher into your application:
+    ```cs
+    public static readonly ITheme BlazingBerry = Themes.BlazingBerry;
+    public static readonly ITheme BlazingDark = Themes.BlazingDark;
+    public static readonly ITheme Purple = Themes.Purple;
+    public static readonly ITheme OfficeWhite = Themes.OfficeWhite;
+    ```
+1. For Fluent themes, call the [Clone()](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxThemeBase-1.Clone(System.Action--0-)) method to add theme stylesheets and change theme mode:
 
-1. Copy this example's [ThemeSwitcher](./CS/switcher/switcher/ThemeSwitcher) folder to your project.
-2. In the *_Imports.razor* file, import the `switcher.ThemeSwitcher` namespace and files located in the *ThemeSwitcher* folder:
+    ```cs
+    public static readonly ITheme FluentLight = Themes.Fluent.Clone(props => {
+        props.AddFilePaths("css/theme-fluent.css");
+    });
+    public static readonly ITheme FluentDark = Themes.Fluent.Clone(props => {
+        props.Mode = ThemeMode.Dark;
+        props.AddFilePaths("css/theme-fluent.css");
+    });
+    ```
+1. For Bootstrap themes, call the [Clone()](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxThemeBase-1.Clone(System.Action--0-)) method to add a Bootstrap theme stylesheet. Use the same approach if you want to apply your own stylesheets.
 
     ```cs
-    @using <YourProjectName>.ThemeSwitcher
-    @using switcher.ThemeSwitcher
+    public static readonly ITheme BootstrapDefault = Themes.BootstrapExternal.Clone(props => {
+        props.Name = "Bootstrap";
+        props.AddFilePaths("https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css");
+        props.AddFilePaths("css/theme-bs.css");
+    });
     ```
+1. Declare an enumeration with themes:

```

Comment:
So this enumeration doesn't seem to be connected to any previous or following code. Maybe I would delete this part... Or we should add some other code that somehow uses this enumerator.